server/memcached/src/main/java/org/infinispan/server/memcached/MemcachedStats.java

```diff
@@ -0,0 +1,16 @@
+package org.infinispan.server.memcached;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * @since 15.0
+ **/
+public class MemcachedStats {
+   public final AtomicLong incrMisses = new AtomicLong();
```

Comment:
It may be worth using an AtomicLongFieldUpdater to reduce the size of this class.

Reply:
Done